### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Daniil-Pankieiev**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -2,4 +2,58 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = [0, 0] if coords is None else coords
```
Comment (prochigor) [inline app/main.py:10]: Do you need else here?

**Review by prochigor — APPROVED** [review]
I approve but look at the comment.

File: app/main.py (hunk 1 above)
Comment (Daniil-Pankieiev) [reply app/main.py:10]: Yes, it wont work without else

File: app/main.py [hunk 2]
```diff
@@ -2,4 +2,58 @@ class Cargo:
...
+    def go_left(self, step: int = 1) -> None:
+        self.coords[0] -= step
+
+    def get_info(self) -> str:
+        return f"Robot: {self.name}, Weight: {self.weight}"
+
+
+class FlyingRobot(BaseRobot):
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        if coords is None:
```
Comment (fsocie7y) [inline app/main.py:30]: do this action as above) in one line
and u can move it in super.__init__()

**Review by fsocie7y — CHANGES_REQUESTED** [review]

**Review by fsocie7y — APPROVED** [review]
well done ^_^

**Pull request closed without merging**